src/language/variables.md

```diff
@@ -44,33 +44,41 @@ The Dart language enforces sound null safety.
 
 Null safety prevents an error that results from unintentional access
 of variables set to `null`. The error is called a null dereference error.
-If you try to access a property or call a method on a variable set to `null`,
-that causes a null dereference error.
+A null dereference error occurs when you access a property or call a method
+on an expression that evaluates to `null`.
+An exception to this rule is when `null` supports the property of method,
+like `toString()` or `hashCode`). With null safety, the Dart compiler
+detects these potential errors at compile time.
+
 For example, say you want to find the absolute value of an `int` variable `i`.
 If `i` is `null`, calling `i.abs()` causes a null dereference error.
 In other languages, trying this could lead to a runtime error,
 but Dart's compiler prohibits these actions.
 Therefore, Dart apps can't cause runtime errors.
 
-Null safety introduces two key changes to type declaration.
+Null safety introduces three key changes:
 
-First, you can specify that a variable _can_ be set to `null`.
-To call out nullability, you add a `?` to the end of the type declaration.
+1.  When you specify a type for a variable, parameter, or another
+    relevant component, you can control whether the type allows `null`.
+    To call out nullability, you add a `?` to the end of the type declaration.
```

Comment:
"call out nullability" -> "enable nullability".

("Call out" is sort of an auto-antonym.)